docs/blog/starboard.md

```diff
@@ -0,0 +1,37 @@
+---
+id: blog_starboard
+title: Improving XKS security using Starboard
+---
+
+Just like for all companies today security is top of mind for Xenit.
+As part of improving Xenit Kubernetes Service (XKS), we have the last few months working on implementing [Aqua Securitys](https://www.aquasec.com/) [Starboard](https://github.com/aquasecurity/starboard) to our clusters.
+
+Thanks to this we will be able to quickly generate dashboards with the amount of critical CVE:s in our clusters. This will make it much easier for us to find if our platform container images or our customers for example is
+having issues with the latest [log4j CVE:s](https://nvd.nist.gov/vuln/detail/CVE-2021-44228).
+
+We use Starboard to continuously scan all container images for running on on XKS, both from a platform point of view but also our customers images.
+In the background Starboard uses [Trivy](https://github.com/aquasecurity/trivy/) to scan the container images. Trivy is the same tool that we use to scan our container images at build time so our XKS customers will have experience reading the Trivy output.
+
+When starting to use Starboard we noticed a few feature that where missing, that we really wanted. And since Starboard is open-source we thought why not help to implement these features.
+
+The first issue we found was that the Starboard operator don't generate any metrics of how many CVE:s that we have per container image.
+Since we want a simple way of visualizing these kind of data in the reports that we generate to our customers every month we started to look for a solution.
```

Comment:
Historically, data was plural of countable datum, but today it is mostly treated as a mass noun, so "this kind of data"